lib/parallel_tests/cli.rb

```diff
@@ -186,6 +186,7 @@ def parse_options!(argv)
         opts.on('--ignore-tags [PATTERN]', 'When counting steps ignore scenarios with tags that match this pattern')  { |arg| options[:ignore_tag_pattern] = arg }
         opts.on("--nice", "execute test commands with low priority.") { options[:nice] = true }
         opts.on("--runtime-log [PATH]", "Location of previously recorded test runtimes") { |path| options[:runtime_log] = path }
+        opts.on("--allowed-missing [FLOAT]", "Allowed percentage of missing runtimes (default = 0.5)") { |percent| options[:allowed_missing] = percent.to_f }
```

Comment:
0.5 is no a percentage ... so let's do 50 ... and convert it later ... and no more .to_f